web-marketplace/src/pages/ProfileEdit/ProfileEdit.ViewProfile.tsx

```diff
@@ -21,9 +21,9 @@ export const ViewProfileButton = ({ setIsWarningModalOpen }: Props) => {
     <OutlinedButton
       onClick={() => {
         if (isDirtyRef?.current) {
-          setIsWarningModalOpen('/profile/portfolio');
+          setIsWarningModalOpen('/profile');
```

Comment:
/profile/portfolio is only available to web3 users which is why it was displaying a warning to connect a wallet address
updating to the generic /profile path